community/modules/scheduler/htcondor-configure/outputs.tf

```diff
@@ -41,6 +59,15 @@ output "execute_point_service_account" {
   ]
 }
 
+output "execute_point_service_account_iam_email" {
```

Comment:
Is it really necessary to expose this parallel channel. It seems like it will just cause user confusion and is complexity we can handle in our modules. It feels like we are pushing the complexity into the interface.

Reply:
Resolving due to lower scope of PR